### PR TITLE
Ps migrations rabbitmq

### PR DESCRIPTION
**Pull request by fullp**
Added RabbitMq implementation to support BTS's calls

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/store/database/entity/external/PostFormField.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "id", unique = true, nullable = false, precision = 64)
+	private Long id;
+
+	@Column(name = "field_name")
+	private String fieldName;
+
+	@Column(name = "external_system_type")
```
Comment (fullp) [inline src/main/java/com/epam/ta/reportportal/store/database/entity/external/PostFormField.java]: fieldType should be string?

File: src/main/java/com/epam/ta/reportportal/core/configs/RabbitMqConfiguration.java [hunk 2]
```diff
@@ -100,6 +106,31 @@ public Queue finishItemQueue() {
...
+        return new Queue(DATA_STORAGE_FETCH_DATA_QUEUE);
+    }
+
+    @Bean
+    public Queue testItemsFindOneQueue() {
+        return new Queue(TEST_ITEMS_FIND_ONE_QUEUE);
+    }
+
+    @Bean
+    public Queue ExternalSystemFindOne() {
```
Comment (avarabyeu) [inline src/main/java/com/epam/ta/reportportal/core/configs/RabbitMqConfiguration.java]: method name starts from lowercase symbol

File: src/main/java/com/epam/ta/reportportal/store/database/mappers/LogMapper.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import static com.epam.ta.reportportal.store.jooq.Tables.LOG;
+
+public class LogMapper {
+
+	public LogMapper() {
+		//static only
+	}
+
+	public static Log getLog(Record record) {
+		Log log = new Log();
```
[the remark this answers is not in the document]
Comment (avarabyeu) [reply src/main/java/com/epam/ta/reportportal/store/database/mappers/LogMapper.java]: We can wrap response from rabbit with some structure like
RS {
 Data data;
}
and send RS with empty/null data
Does it answer the question?

[the remark this answers is not in the document]
Comment (fullp) [reply src/main/java/com/epam/ta/reportportal/store/database/mappers/LogMapper.java]: I mean if record is null while getLog method execute, should we throw some exception or something else?

**Pull request merged**